mlflow/store/artifact/azure_blob_artifact_repo.py

```diff
@@ -60,20 +60,21 @@ def parse_wasbs_uri(uri):
         parsed = urllib.parse.urlparse(uri)
         if parsed.scheme != "wasbs":
             raise Exception("Not a WASBS URI: %s" % uri)
-        match = re.match("([^@]+)@([^.]+)\\.blob\\.core\\.windows\\.net", parsed.netloc)
+        match = re.match("([^@]+)@([^.]+)\\.(blob\\.core\\.(windows\\.net|chinacloudapi\\.cn))", parsed.netloc)
         if match is None:
             raise Exception(
-                "WASBS URI must be of the form " "<container>@<account>.blob.core.windows.net"
+                "WASBS URI must be of the form " "<container>@<account>.blob.core.windows.net" " or <container>@<account>.blob.core.chinacloudapi.cn"
             )
         container = match.group(1)
         storage_account = match.group(2)
+        api_uri = match.group(3)
```

Comment:
```suggestion
        api_uri_suffix = match.group(3)
```

Can we add `_suffix` here?

Reply:
added.